py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/__init__.py

```diff
@@ -4,16 +4,25 @@
 """
 
 from .batcher import batch_files
-from .config_base import ConfigType
-from .configuration import Configuration
-from .convert import gz_to_pyarrow
-from .error import ArgumentException
-from .error import ConfigTypeFromFilenameException
-from .error import NoImplException
-from .s3_utils import file_list_from_s3
-from .s3_utils import move_s3_objects
-from .lambda_types import LambdaDict
-from .lambda_types import LambdaContext
+from .converter import (
```

Comment:
i switched to this import style for init because it felt clearer to me what we're bringing in.